Apps/Admin/Tests/Functional/cypress/integration/e2e/support.cy.js

```diff
@@ -139,4 +140,18 @@ describe("Support", () => {
 
         cy.log("Verify email returns no results test finished.");
     });
+
+    it("Verify dependents query returns results.", () => {
+        cy.log("Verify dependents query returns results test started.");
+
+        cy.get("[data-testid=query-type-select]").click({ force: true });
+        cy.get("[data-testid=query-type]")
+            .contains("Dependent")
+            .click({ force: true });
+        cy.get("[data-testid=query-input]").clear().type(dependentPhn);
+        cy.get("[data-testid=search-btn]").click();
+        cy.get("[data-testid=user-table]").find("tr").should("have.length", 2);
+
+        cy.log("Verify dependents query returns results test finished.");
```

Comment:
We don't need these sort of log statements at the start and end of functional tests, as that information is readily available when looking at the test runs.  We usually only want log statements in larger tests that have multiple parts to help determine what point we are at and document the behaviour.

I see the other tests in the file follow the same logging pattern as this one and I suggest we do away with those log statements as well.

Reply:
removed all the log statements